### DISCUSSION TITLE
【转发扩散帮她回家】谁的娃丢了？！

### DISCUSSION THREAD

**Discussion by UAreLoved** [share121/inter-knot#2854] [其他]
<img width="817" alt="截屏2024-08-18 22 33 02" src="https://github.com/user-attachments/assets/2c0f07d9-7aba-4ce6-a6b2-426927097cdf">

今天在报刊亭买刮刮卡，就看到一个小妹妹在等谁的样子。问她在等谁，家住哪儿，都不回答我555暂时让报刊亭的奶奶看着了，也已经联系治安局，但是我还要打工没办法守着（原谅打工人），要是父母看到，快点把娃领回去吧！！

**Comment by Kirolei** [comment]
这个委托我接了！

**Comment by yssxrc** [comment]
这个委托我抢了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by bearfish74** [comment]
可爱的小孩！！

**Comment by aizai1173** [comment]
美味的小孩...已经被我7啦！！！

**Reply by UAreLoved** [reply]
治安局，open the door！
